apps/backend-e2e/src/map-review.e2e-spec.ts

```diff
@@ -331,6 +334,41 @@ describe('Map Reviews', () => {
           token: token
         }));
 
+      it('should 503 if the map-review key is true', async () => {
+        const [admin, adminToken] = await db.createAndLoginUser({
+          data: { roles: Role.ADMIN }
+        });
+
+        await req.patch({
+          url: 'admin/killswitch',
+          status: 204,
+          body: {
+            MAP_REVIEWS: true
+          },
+          token: adminToken
+        });
+
+        const response = await req.del({
+          url: `map-review/${review.id}`,
+          status: 503,
+          token: token
+        });
+
+        expect(response.body.statusCode).toEqual(503);
```

Comment:
Unnecessary, the `status: 503` check in `req.del` is doing exactly this under the hood.

Reply:
Applies to loads below